packages/app/src/app/pages/Dashboard/Components/Sandbox/SandboxCard.tsx

```diff
@@ -136,16 +137,25 @@ const SandboxTitle: React.FC<SandboxTitleProps> = React.memo(
 
 type SandboxStatsProps = {
   isFrozen?: boolean;
+  username: string;
 } & Pick<
   SandboxItemComponentProps,
   'noDrag' | 'timeAgo' | 'PrivacyIcon' | 'sandbox' | 'restricted'
 >;
 const SandboxStats: React.FC<SandboxStatsProps> = React.memo(
-  ({ isFrozen, noDrag, timeAgo, PrivacyIcon, sandbox, restricted }) => {
+  ({
+    username,
+    isFrozen,
+    noDrag,
+    timeAgo,
+    PrivacyIcon,
+    sandbox,
+    restricted,
+  }) => {
     const boxType = sandbox.isV2 ? 'devbox' : 'sandbox';
     const timeAgoText = (
       <Text size={12} truncate>
-        {shortDistance(timeAgo)}
+        {shortDistance(timeAgo)} by {username}
```

Comment:
Oh, I was under the impression this would list the author, not the last person who accessed it (that should be the customer's expectation as well). Can we update it?

Reply:
Yeah it lists the creator’s name, but _anyone_’s timestamp. So if i look at it but you created it, it would say “ yesterday by Filipe” even if you didn’t open it for weeks - that’s what i meant could be confusing.

Reply:
This could be confusing indeed. Can we list the creation date instead? Maybe we can also list relevant events like these (created on; created by; last accessed on; last accessed by) on the "Sandbox Info" panel.